src/krylov/gmres.h

```diff
@@ -59,7 +59,8 @@ typedef struct
    HYPRE_Int    (*Matvec)        ( void *matvec_data, HYPRE_Complex alpha, void *A,
                                    void *x, HYPRE_Complex beta, void *y );
    HYPRE_Int    (*MatvecDestroy) ( void *matvec_data );
-   HYPRE_Real   (*InnerProd)     ( void *x, void *y );
+   HYPRE_Int    (*InnerProd)     ( void *x, void *y, HYPRE_Int *num_tags_ptr,
```

Comment:
Shouldn't this be a new function name "InnerProdTagged"? Is the plan to overload the InnerProd function name?

Reply:
That's more accurate, but I was initially planning to abuse the nomenclature here... I'm fine with changing the name